src/Functions/array/arrayAUCUnscaled.cpp

```diff
@@ -0,0 +1,212 @@
+#include <DataTypes/DataTypesNumber.h>
```

Comment:
Please see my other comment, we could avoid duplicating 200+ LOC by slightly extending the existing `arrayAUC` function.

Reply:
agreed, will do this change